src/taipy/_cli/_migrate.py

```diff
@@ -0,0 +1,487 @@
+    """
+    conn.execute(query)
+    conn.commit()
+
+
+def __insert_job(job: dict, conn):
+    query = f"""
+        INSERT INTO job (id, task_id, status, force, submit_id, submit_entity_id, creation_date, subscribers, stacktrace, version)
+        VALUES ({job['id']}, {job['task_id']}, {job['status']}, {job['force']}, {job['submit_id']}, {job['submit_entity_id']}, {job['creation_date']}, {job['subscribers']}, {job['stacktrace']}, {job['version']})
+    """
+    conn.execute(query)
+    conn.commit()
+
+
+def __insert_cycle(cycle: dict, conn):
+    query = f"""
+        INSERT INTO scenario (id, name, frequency, properties, creation_date, start_date, end_date)
+        VALUES ({cycle['id']}, {cycle['name']}, {cycle['frequency']}, {cycle['properties']}, {cycle['creation_date']}, {cycle['start_date']}, {cycle['end_date']})
+    """
+    conn.execute(query)
+    conn.commit()
+
+
+def __insert_version(version: dict, conn):
+    query = f"""
+        INSERT INTO version (id, config_id, creation_date, is_production, is_development, is_latest)
+        VALUES ({version['id']}, {version['config_id']}, {version['creation_date']}, {version['is_production']}, {version['is_development']}, {version['is_latest']})
+    """
+    conn.execute(query)
+    conn.commit()
+
+
+def __write_entities_to_sql(_entities: Dict, _versions: Dict, db_file: str = "test.db"):
+    conn = sqlite3.connect(db_file)
+
+    for k, entity in _entities.items():
+        if "SCENARIO" in k:
+            __insert_scenario(entity["data"], conn)
+        elif "TASK" in k:
+            __insert_task(entity["data"], conn)
+        elif "DATANODE" in k:
+            __insert_datanode(entity["data"], conn)
+        elif "JOB" in k:
+            __insert_job(entity["data"], conn)
+        elif "CYCLE" in k:
+            __insert_cycle(entity["data"], conn)
+
+    for k, version in _versions.items():
+        __insert_version(version, conn)
+
+
+FCT_MIGRATION_MAP = {
+    "SCENARIO": _migrate_scenario,
+    "TASK": _migrate_task_entity,
+    "DATANODE": _migrate_datanode_entity,
+    "JOB": _migrate_job,
+    "VERSION": _migrate_version,
+}
+
+
+def _migrate_fs_entities(path: str) -> None:
+    __logger.info("Starting entity migration")
+    entities = _load_all_entities_from_fs(path)
+
+    __logger.info("Migrating SCENARIOS")
+    entities = _migrate_entity("SCENARIO", entities)
+
+    __logger.info("Migrating TASKS")
+    entities = _migrate_entity("TASKS", entities)
+
+    __logger.info("Migrating DATANODES")
+    entities = _migrate_entity("DATANODE", entities)
+
+    __logger.info("Migrating JOBS")
+    entities = _migrate_entity("JOB", entities)
+
+    __logger.info("Migrating VERSION")
+    entities = _migrate_entity("VERSION", entities)
```

Comment:
This code block is duplicated on all `_migrate_..._entities()`. Probably should split it to a dedicated method.
```suggestion
    __migrate_entities(entities)
```